registry/eip155/tron-evm.json

```
@@ -0,0 +1,39 @@
+{
+  "id": "tron-evm",
+  "shortName": "TRON EVM",
+  "fullName": "TRON EVM Mainnet",
+  "aliases": ["tronevm", "tron-evm-mainnet", "evm-728126428"],
+  "caip2Id": "eip155:728126428",
+  "graphNode": { "protocol": "ethereum" },
+  "explorerUrls": ["https://tronscan.org"],
+  "rpcUrls": [
+    "https://api.trongrid.io/jsonrpc",
+    "https://tron.drpc.org",
+    "https://rpc.ankr.com/tron_jsonrpc"
+  ],
+  "relations": [{ "kind": "evmOf", "network": "tron" }],
+  "apiUrls": [
+    { "url": "https://apilist.tronscanapi.com/api", "kind": "etherscan" }
+  ],
+  "networkType": "mainnet",
+  "services": {
+    "subgraphs": [],
+    "sps": [],
+    "firehose": ["mainnet-evm.tron.streamingfast.io:443"],
+    "substreams": ["mainnet-evm.tron.streamingfast.io:443"]
+  },
+  "issuanceRewards": true,
```

```suggestion
  "issuanceRewards": false,
```
hasn't been approved for indexing rewards